docs/references/backend/authenticate-request.mdx

```diff
@@ -200,6 +223,33 @@ It is recommended to set these options as [environment variables](/docs/deployme
   </Tab>
 </Tabs>
 
+### Machine authentication
+
+By default, `authenticateRequest()` will authenticate a session request. To authenticate a machine request, you need to set the `acceptsToken` option to a machine token type, such as `'api_key'`, `'oauth_token'` or `'machine_token'`.
+
+```tsx
+import { createClerkClient } from '@clerk/backend'
+
+export async function GET(request: Request) {
+  const clerkClient = createClerkClient({
+    secretKey: process.env.CLERK_SECRET_KEY,
+    publishableKey: process.env.CLERK_PUBLISHABLE_KEY,
```

Comment:
I don't think you need publishable key for backend? https://clerk.com/docs/references/backend/overview#usage

Reply:
Yes, we can skip the PK here